data/utils/distances_mpg.json

```diff
@@ -0,0 +1,39 @@
+{
+    "mpg_curved": 0.021429823922273755,
+    "mpg_2": 0.6693008207479366,
+    "mpg_25": 0.9490503396238478,
+    "mpg_13": 0.9958652349320338,
+    "mpg_8": 0.6279256278953644,
+    "mpg_18": 1.0807677647324,
+    "mpg_growth": 0.015046932082086824,
+    "mpg_14": 1.136393455012086,
+    "mpg_17": 0.6629682036321781,
+    "mpg_15": 0.39499263265524126,
+    "mpg_1": 0.5248772252922564,
+    "mpg_neutrinos": 0.04574273467040171,
+    "mpg_16": 0.3951709821680723,
+    "mpg_28": 1.2402690859168137,
+    "mpg_central": 0.0,
+    "mpg_29": 1.1953808151964511,
+    "mpg_10": 0.5141590554595118,
+    "mpg_19": 0.4918722959301434,
+    "mpg_21": 0.6074099216943172,
+    "mpg_seed": 0.006530450163938471,
```

Comment:
I was expecting seed to be the same as central, do you know why is it different?

Reply:
mmh not sure, the difference is very small. I'll check.

Reply:
The cosmo params are exactly the same, but central and seed have small differences in the IGM parameters